cmd/storage-rpc-client.go

```diff
@@ -143,13 +143,15 @@ const maxAllowedNetworkIOError = 1024
 // Initializes the remote RPC connection by attempting a login attempt.
 func (n *networkStorage) Init() (err error) {
 	// Attempt a login to reconnect.
-	return n.rpcClient.Login()
+	err = n.rpcClient.Login()
+	return toStorageErr(err)
```

Comment:
Which errors are meaningfully converted using ``toStorageErr`` in the case of Login?